_includes/banner.html

```diff
@@ -34,7 +34,6 @@
           <img
             class="usa-banner__icon usa-media-block__img"
             src="{{ site.baseurl }}/assets/img/icon-dot-gov.svg"
-            role="img"
```

Comment:
Not sure why this is being removed. It was following canonical source for the design system component.

https://designsystem.digital.gov/components/banner/

(Address before merge)

Reply:
I bet that's a merge conflict gone wrong... I had to address some merge conflicts with similar roles elsewhere: https://github.com/18F/identity-site/pull/826/commits/6da8d47684914349801d3fd9d4c59984c98148c1

Reply:
PR to fix: https://github.com/18F/identity-site/pull/864